distribution/test/test-rest-endpoint/pom.xml

```diff
@@ -40,9 +40,9 @@
     </build>
     <dependencies>
         <dependency>
-            <groupId>javax.ws.rs</groupId>
-            <artifactId>javax.ws.rs-api</artifactId>
-            <version>2.1</version>
+            <groupId>jakarta.ws.rs</groupId>
+            <artifactId>jakarta.ws.rs-api</artifactId>
+            <version>2.1.6</version>
```

Comment:
should we use ${javax.ws.rs.version} here?

Reply:
yeah, I'll change it